meta/state.go

```diff
@@ -62,6 +64,16 @@ func (r *localRaft) remove() error {
 	return nil
 }
 
+func (r *localRaft) shutdown() error {
```

Comment:
Ok, I understand what you are saying.  Removed `shutdown` and now use `close` as intended.  Thanks!